terraform-aws-github-runner/modules/runner-binaries-syncer/lambdas/runner-binaries-syncer/package.json

```diff
@@ -31,6 +29,9 @@
     "typescript": "^4.2.3"
   },
   "dependencies": {
+    "@aws-sdk/client-s3": "^3.879.0",
+    "@aws-sdk/lib-storage": "^3.879.0",
+    "@vercel/ncc": "^0.38.3",
```

Comment:
was getting build problems so I also swapped the ncc dependency. the other lambdas use `@vercel`, maybe not this version tho

Reply:
This is more than likely correct